recipes/Vegan Carrot Cake.qmd

```diff
@@ -0,0 +1,34 @@
+title: "Template for new recipe"
+image: "https://biancazapatka.com/en/moist-and-simple-carrot-cake-vegan/"
+image-alt: "Add the URL to the image here"
+categories:
+  - Add a list of catgeories here
+---
+
+## {{< fa basket-shopping >}} Ingredients
+
+- [ ] Add your ingredients here
+- [1] 250g Carrots
```

Comment:
Please remove the numbers from the To-Do List items. It should just contain a whitespace:

```r
- [ ] 250g Carrots
```